src/widgets/mentorship-courses/mentorship-courses.test.tsx

```diff
@@ -1,9 +1,9 @@
 import { screen, within } from '@testing-library/react';
 
-import { MentorshipCourseTitles } from '../../../../../dev-data/mentorship-data.types';
+import { MentorshipCourseTitles } from '../../../dev-data/mentorship-data.types';
```

Comment:
we can make it easier
```suggestion
import { MentorshipCourseTitles } from 'data';
```